### PR TITLE
Change import to relative

### PR DESCRIPTION
**Pull request by LinboLen**
Issue Number: N/A

## PR Details

nest cli build app will auto change absolute to relative.
but when use swc or tsc to build app will cause problem when run
directly by node.

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by arielweinberger** [comment]
Related to #3313 

We should aim to have only relative imports.

File: packages/amplication-server/src/core/auth/auth.module.ts [hunk 1]
```diff
@@ -57,7 +57,8 @@ import { GitHubAuthGuard } from './github.guard';
         );
         const options = await githubConfigService.getOptions();
         if (options === null) {
-          return;
+			// @ts-ignore
```
Comment (tupe12334) [inline packages/amplication-server/src/core/auth/auth.module.ts:60]: Why did you add @ts-ignore?

**Comment by abrl91** [comment]
@LinboLen Thank you and congrats on your first contribution.
Please resolve the merge conflict.

**Comment by arielweinberger** [comment]
I have resolved it. Let's address the `@ts-ignore` statement and merge.

**Review by arielweinberger — APPROVED** [review]
